src/rmp/README.md

```diff
@@ -53,6 +53,21 @@ restructure
 | `-abc_logfile` | Output file to save abc logs to. |
 | `-work_dir` | Name of the working directory for temporary files. If not provided, `run` directory would be used. |
 
+### Resynth
+
+Resynthesis can be performed with the resynthesis command it will attempt to drive
+all paths to zero slack. It's an improved version of restructure. If you have multiple
+corners this command with use what's set in `sta::cmd_corner` and can be set with `sta::set_cmd_corner`
```

Comment:
...command _will_ use...?

We don't encourage people to use commands in namespace.  Why is set_cmd_corner necessary?  Could it be a command arg?

Reply:
Done.